src/content/ui/animations/staggered-animations.md

```diff
@@ -339,6 +326,46 @@ class _StaggerDemoState extends State<StaggerDemo>
 }
 ```
 
+## Review a staggered pic selection example
+
+This complex example uses the code from [staggered_pic_selection][].
+It displays a list of images in one of three sizes.
+
+{% render docs/captioned-image.liquid,
+image:"ui/animations/staggered/no-pic-selected.png",
+caption:"The **staggered_pic_animation** example in macOS" %}
+
+The animation uses two [animation controllers][]:
+
+1. One animates the selection or deselection an image.  
+   This animation is staggered.
+   (To see this effect,
+   you might need to increase the `timeDilation` value.)
+
+1. Another animates the deletion of an image.
+
+When you run the app, the animation renders when you click an image.
+
+1. Select one of the largest images.  
+   It shrinks as it displays a checkmark inside a purple circle.
+
+    {% render docs/captioned-image.liquid,
+    image:"ui/animations/staggered/large-pic-selected.png",
+    imgstyle:"w50p",
+    caption:"A large image selected in the example app" %}
+
+1. Select one of the smallest images.
+   The large image expands as the checkmark disappears.
+   Before the large image has finished expanding,
+   the small image shrinks to display its checkmark.
+
+    {% render docs/captioned-image.liquid,
+    image:"ui/animations/staggered/small-pic-selected.png",
+    imgstyle:"w50p",
+    caption:"A small image selected in the example app" %}
+
+This staggered behavior resembles the behavior of Google Photos.
```

Comment:
Keep original wording:
```suggestion
This staggered behavior is similar to what you might see in Google Photos.
```

Reply:
Trying to write in active voice per [GSG](https://developers.google.com/style/voice).